daisy_workflows/image_import/windows/translate_bootstrap.ps1

```diff
@@ -193,7 +194,13 @@ try {
   else {
     Copy-32bitPackages
   }
-  Run-Command bcdboot "${script:os_drive}\Windows" /s $bcd_drive
+  if ($partition_style -eq "MBR") {
+    Write-Output 'MBR partition detected. Resetting bootloader.'
```

Comment:
Should we tag this so it can be picked up by import wrapper so it can modify the workflow and add UEFI_COMPATIBLE? Sync up with Tony on this.

Reply:
discussed with Romain. This one is fine. I will modify the latter one in the next PR. (only return info for UEFI case).